stdlib/LinearAlgebra/src/lbt.jl

```diff
@@ -103,6 +103,38 @@ struct LBTConfig
     end
 end
 
+Base.show(io::IO, lbt::LBTLibraryInfo) = print(io, "LBTLibraryInfo($(basename(lbt.libname)))")
+function Base.show(io::IO, mime::MIME{Symbol("text/plain")}, lbt::LBTLibraryInfo)
+    summary(io, lbt); println(io)
+    println(io, "├ Library: ", basename(lbt.libname))
+    println(io, "├ Interface: ", lbt.interface)
+      print(io, "└ F2C: ", lbt.f2c)
+end
+
+function Base.show(io::IO, lbt::LBTConfig)
+    if length(lbt.loaded_libs) <= 3
+        print(io, "LBTConfig(", join(basename.(getfield.(lbt.loaded_libs, :libname)), ", "), ")")
```

Comment:
For this fallback method, can we get the interface type printed out as well?

Reply:
Done.

```Julia
julia> print(lbt)
LBTConfig([ILP64] libopenblas64_.dylib)

julia> print(lib)
LBTLibraryInfo(libopenblas64_.dylib, ilp64)
```